README.md

```diff
@@ -82,15 +82,15 @@ plugins=(
 )
 ```
 
-_Note that the plugins are separated by whitespace. **Do not** use commas between them._
+_Note that the plugins are separated by whitespace (spacebar/enterkey). **Do not** use commas between them or it will break._
 
 #### Using Plugins
 
-Most plugins (should! we're working on this) include a __README__, which documents how to use them.
+Most plugins (should! we're working on this) include a __README__, which documents how to use them. This README should show the aliases and extra goodies that are inclued in that particular plugin.
```

Comment:
Some plugins don't add aliases, can this be more generic? Also we can get rid of the `should! we're working on this` part. It is a statement: each plugin has its own README documenting it, we finally got to having one for each.

Reply:
Fixed, please check and make sure that I worded it properly.